newrelic.ini

```diff
@@ -188,34 +228,26 @@ error_collector.ignore_errors = app.v2.errors:BadRequestError jsonschema.excepti
 # specific environment will be used when the environment argument to the
 # newrelic.agent.initialize() function has been defined to be either
 # "development", "test", "staging" or "production".
+#
 
-# If this setting is enabled, it will capture package and version
-# information on startup of the agent that is displayed in the APM
-# environment tab.
-# In applications that have a large number of packages, having this
-# setting enabled may cause a CPU spike as it captures all the package
-# and version information. It is recommended in those cases to disable
-# this setting.
-# Disabling this setting will disable the ability to detect vulnerabilities in outdated packages.
+app_name = notification-lambda-api-environment-not-set
```

Comment:
I like using a custom noisy name here to know the changes we are doing are coming through.